### ISSUE TITLE
Why train dataset are smaller than the original redial dataset?

### ISSUE THREAD

**Issue by icedpanda** [JD-AI-Research-NLP/RevCore#10]
Hi, thanks for sharing the source code.

I found that the train/valid/test data are slightly different from the original `Redia Dataset`.

The original `redial dataset` has a total of **11348** dialogues, **10006** for training and **1342** for testing.  See [here](https://redialdata.github.io/website/datasheet)

However, the provided training file only has **8104** samples under the `data` folder. And the validation set has **901** samples, resulting in a  total of **9005** samples.  (train+valid+test = 10347)

Initially, I thought this happened due to the `recommendation splits` from the redial dataset as the `test set` was generated at a later time. And papers like KBRD, and KGSF all stated a total of 10006 samples. (not train but total) Hence I thought you may split `training` to `train`/`valid`/`test` and this would give us roughly 80/10/10 splits. (8104/10006 = 80.9%).

However, I found that `test_data.jsonl` under the `data` folder is the same as in the original redial. Therefore, my assumption seems not to stand.  

To me, I feel like its still doing the `train`/`val`/`test` split on the original `train` data and keeping the original `test` as the final `test` data. But somehow `train_valid` or `train_test` is dumped and resulting these `train_train`, `train_valid`, `test` files. (Here just assume train_test is dumped)

I am wondering what happened to those 10% data. And I found this is happening in all relevant repos like KBRD/KGSF/CRSLab.

Thanks in advance

**Comment by luyulalalala** [comment]
Thank you for bringing up this problem.  I checked data of KBRD/KGSF/CRSLab and the newest Redial, and agree with your guess. However, for a fair comparison, I need to follow the data used by those baselines.

**Comment by icedpanda** [comment]
Hi @luyulalalala, thanks for the prompt response. I guess I need to confirm with repo then
